addons/main/functions/fnc_updateCtrlPosition.sqf

```diff
@@ -0,0 +1,24 @@
+#include "script_component.hpp"
+params ["_unit", "_ctrlGroup"];
+private _stanceDisplay = uiNamespace getVariable [QGVAR(stanceDisplay), displayNull];
+if (isNull _stanceDisplay) then {
+    private _guiDisplay = uiNamespace getVariable ["IGUI_displays", []];
+    private _index = _guiDisplay findIf {(str _x) == "Display #303"};
+    if (_index isNotEqualTo -1) then {
+        _stanceDisplay = _guiDisplay select _index;
+        uiNamespace setVariable [QGVAR(stanceDisplay), _stanceDisplay];
+    };
+};
```

Comment:
Can this be moved to postinit, or why is this here?

Reply:
https://github.com/diwako/armor_plates_system/blob/e7c537d6ba442c0dc49af8fe35886583f093ed7b/addons/main/functions/fnc_initPlates.sqf#L2

Reply:
Does not really answer the question. The question was why is the search for the stance display here and not in init plates of the like?